zerver/tests/test_messages.py

```diff
@@ -30,7 +30,7 @@ def test_presence_idle_user_ids(self) -> None:
         othello_alerted = False
         othello_notifications_data = self.create_user_notifications_data_object(user_id=othello.id)
 
-        def assert_missing(user_ids: List[int]) -> None:
+        def assert_returned_list(user_ids: List[int]) -> None:
```

Comment:
Why not call this `assert_active_presence_idle_user_ids`?

Reply:
Makes sense, done!